src/blog/_posts/2023-08-31-community-spotlight-august-2023.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Cadence Community Spotlight Update - August 2023
+
+date: 2023-08-31
+author: Sharan Foga
+authorlink: https://www.linkedin.com/in/sfoga/
+---
+Welcome to the latest of our regular monthly Community Spotlight updates that gives you news from in and around the Cadence community!
+
+Please see below for a roundup of the highlights:
+
+## More Cadence How To's ##
+
+You might have noticed that we have had a few more contributions to our blog from [Chris Qin](https://www.linkedin.com/in/chrisqin0610). Chris has been busy sharing insights, and tips on a few important Cadence topics. The objective it to help the community with any potential problems.
```

Comment:
He is also rewriting the samples and making sure they are in working state. We can mention that.

Reply:
Updated. And thanks